source/sysconf/systemd.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sysconf
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+
+func runCmd(cmd string, args []string) (string, error) {
+	if len(cmd) < 1 {
+		return "", fmt.Errorf("Need command")
+	}
+
+	output, err := exec.Command(cmd, args...).CombinedOutput()
+
+	return strings.TrimRight(string(output), "\n"), err
+}
+
+func SystemctlStatus(services string) (success bool) {
```

Comment:
`SystemctlStatusIsActive` might be a more appropriate considering the scope of this function.